client/reducers/action.js

```diff
@@ -68,12 +62,6 @@ export const addTileInfo = (tileInfo) => {
   }
 }
 
-export const levelWon = () => {
-  return {
-    type: LEVEL_WON
-  }
```

Comment:
this is being used in the wincontainer better to dispatch levelWon() in the action file